src/algebra/category/Algebra/limits.lean

```diff
@@ -42,24 +42,24 @@ def sections_subalgebra (F : J ⥤ Algebra R) :
 
 
 instance limit_semiring (F : J ⥤ Algebra R) :
```

Comment:
I wonder if it would be clearer if we added the `.{v}` annotation to _every_ appearance of `Algebra`, rather than just the fraction were otherwise Lean gets confused?